zio-http/src/main/scala/zhttp/http/Response.scala

```diff
@@ -86,7 +81,7 @@ final case class Response private (
 
     val jHeaders = self.getHeaders.encode
     val jContent = self.data match {
-      case HttpData.Text(text, charset) => Unpooled.wrappedBuffer(text.getBytes(charset))
+      case HttpData.Text(text, charset) => Unpooled.copiedBuffer(text, charset)
```

Comment:
Is this change required?

Reply:
@tusharmath 
wrappedbuffer doesn't take in CharSequence .
we can do text.toString.getBytes instead .
Is that preferred?
String.getBytes performs a copy either way

Reply:
Got it. May be we can do something like this.

```scala
case HttpData.Text(ascii: AsciiString, _) => Unpooled.wrappedBuffer(ascii)
case HttpData.Text(text, charset)         => Unpooled.copiedBuffer(text, charset)
```

The overall idea is to reduce the copies we create as much as possible. This makes a big difference in performance when we send large payloads to the clients.

Reply:
ok , ill look into it